### PR TITLE
Add sensor mediation HAL

### PR DESCRIPTION
**Pull request by JaikrishnaNemallapudi**
Initial Release code

Tracked-On: OAM-91265
Signed-off-by: Jaikrishna Nemallapudi <nemallapudi.jaikrishna@intel.com>

### CODE REVIEW

**Review by rbabub — APPROVED** [review]
Looks good to me.

File: sensor_hal.cpp [hunk 1]
```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright Copyright (c) 2020 Intel Corporation
```
Comment (JeevakaPrabu) [inline sensor_hal.cpp]: Remove 1 Copyright

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

File: sensor_hal.cpp (hunk 1 above)
Comment (JaikrishnaNemallapudi) [reply sensor_hal.cpp]: Fixed the duplicate string

**Review by rbabub — APPROVED** [review]
Looks good to me.

**Pull request merged**